packages/conform-valibot/package.json

```
@@ -0,0 +1,66 @@
+{
+	"name": "@conform-to/valibot",
+	"description": "Conform helpers for integrating with Valibot",
+	"homepage": "https://conform.guide",
+	"license": "MIT",
+	"version": "0.0.0",
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"types": "./dist/index.d.ts",
+	"files": [
+		"dist"
+	],
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"module": "./dist/index.mjs",
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js",
+			"default": "./dist/index.mjs"
+		}
+	},
+	"scripts": {
+		"build:js": "rollup -c",
+		"build:ts": "tsc --project ./tsconfig.build.json",
+		"build": "pnpm run \"/^build:.*/\"",
+		"dev:js": "pnpm run build:js --watch",
+		"dev:ts": "pnpm run build:ts --watch",
+		"dev": "pnpm run \"/^dev:.*/\"",
+		"test": "vitest",
+		"prepare": "pnpm run build"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/edmundhung/conform",
+		"directory": "packages/conform-valibot"
+	},
+	"bugs": {
+		"url": "https://github.com/edmundhung/conform/issues"
+	},
+	"dependencies": {
+		"@conform-to/dom": "workspace:*"
```

I think we will need to use a [Caret Ranges](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004) here as we will not be bumping the valibot package version together with `@conform-to/dom` everytime. The caret range should allow people to use a newer version of `@conform-to/dom` without issue:

```suggestion
		"@conform-to/dom": "workspace:^"
```